### PR TITLE
Add common components

### LINKED ISSUE #4: Move core components for `common_resources` to this repo

### PR DESCRIPTION
**Pull request by rcannood**
close #4

### CODE REVIEW

**Comment by rcannood** [comment]
Creating a draft pull request so I don't forget it exists

**Review by KaiWaldrant — APPROVED** [review]
LGTM!

File: .github/workflows/viash-build.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+          The version of the project to build. Example: `1.0.3`.
+          
+          If not provided, a development build with a version name
+          based on the branch name will be built. Otherwise, a release
+          build with the provided version will be built.
+        required: false
+
+jobs:
+  build:
+    uses:  viash-io/viash-actions/.github/workflows/build.yaml@add-working-dir-input
```
Comment (rcannood) [inline .github/workflows/viash-build.yaml:19]:
```suggestion
    uses: viash-io/viash-actions/.github/workflows/build.yaml@add-working-dir-input

**Comment by rcannood** [comment]
still need to change  `@add-working-dir-input` into `@v6` once the updated actions are released; but I will merge this PR already so I can test the build action in the @add-working-dir-input branch.

**Pull request merged**